provision/examples/gardener/azure/main.go

```diff
@@ -40,19 +41,16 @@ func main() {
 			"target_secret":          *secret,
 			"disk_type":              "Standard_LRS",
 			"workercidr":             "10.250.0.0/19",
-			"vnetcidr":               "10.250.0.0/19",
+			"vnetcidr":               "10.250.0.0/16",
 			"worker_max_surge":       4,
 			"worker_max_unavailable": 1,
 			"worker_maximum":         4,
 			"worker_minimum":         2,
 			"machine_image_name":     "coreos",
 			"machine_image_version":  "2303.3.0",
-			"networking_nodes":       "10.250.0.0/19",
-			"networking_pods":        "100.96.0.0/11",
-			"networking_services":    "100.64.0.0/13",
 			"networking_type":        "calico",
 			"service_endpoints":      []string{""},
-			"zone":                   []string{"1"},
+			"zones":                  []string{},
```

Comment:
why do you remove the zone from here ?

Reply:
Because otherwise you have to specify `networking_pods` for the gardener validation to happen.

The interesting thing is if I do not specify it, gardener assigns the same value to it on the server side, so I guess the server side validation has a bug.

This way I just keep the example more simple.

Reply:
Mystery solved, please see comment in https://github.com/kyma-incubator/hydroform/issues/104